OSSM-8759 Pre-migration checklist for product docs (#88198)
openshift/openshift-docs: modules/ossm-migrating-premigration-checklists-using-cert-manager.adoc

// Module included in the following assemblies:
//
// * service-mesh-docs-main/migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc

:_mod-docs-content-type: PROCEDURE
[id="ossm-migrating-premigration-checklists-using-cert-manager_{context}"]

> = Check if you are using the cert-manager tool with your deployment

Reviewer: Please see the mod docs guide on how to format procedure titles. For consistency, it would be best to reformat your filename and id to match the updated title.

> It is important to know if you are using the cert-manager tool with {SMProduct} 2. There are a series of configuration steps you must perform before you can start migrating your deployments and workloads.

Reviewer: ```suggestion
You must check if your deployment uses the cert-manager tool. If you are using the cert-manager tool with {SMProduct} 2, you must perform additional configuration before you can start migrating your deployments and workloads. 
```

.Procedure

. Use one of the following ways to check if you are using the cert-manager tool with your deployment:

> .. Inspect your `ServiceMeshControlPlane` resource:

Reviewer: ```suggestion
.. Inspect your `ServiceMeshControlPlane` resource to verify that the `spec.security.certificateAuthority.type` parameter is set to `cert-manager`:
```

+
[source,yaml]
----

> spec:

Reviewer: You must include YAML front matter in YAML examples: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields

  security:
    certificateAuthority:
      ...
      type: cert-manager
----

> .. Run the following command:

Reviewer: ```suggestion
.. Run the following command to ... :
```
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command

+
[source,yaml]
----
oc get smcp <smcp-name> -n <smcp-namespace> -o jsonpath='{.spec.security.certificateAuthority.type}'
----
+

> .Output

Reviewer: ```suggestion
.Example output
```
